app/src/main/java/ui/SplashActivity.kt

```diff
@@ -318,27 +309,22 @@ class SplashActivity : BaseActivity() {
         }
     }
 
-    /**
-     * Updates the progress [message]
-     */
     private fun updateProgress(message: String) {
         launch(uiDispatcher) {
             progressText.text = message
         }
     }
 
     private suspend fun openHomePage() {
+        val context = this
         withContext(uiDispatcher) {
-            startActivity(Intent(this@SplashActivity, homePageManager.activity))
+            startActivity(Intent(context, homePageManager.activity))
```

Comment:
Personally, I don't see `this@...` as a problem. If you are seeing this a lot, you may as well add `inline val context: get() = this`, `val context = this`, or `lateinit var context: Context` if it doesn't leak the constructor

Reply:
I'm having doubts about removing labelled expressions as well. I may just turn that check off...

Reply:
I've turned that check off. I'll revert this change (and some others maybe).